db/models/api.models.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs')
+
+async function fetchEndpoints () {
+    const data = await fs.promises.readFile('endpoints.json', (err, data) => {
+        return data;
+    });
+    
+    return JSON.parse(data)
```

Comment:
fine, but can just be required in an sent In the controller.  Will save you some lines of code!